src/main/java/ca/corefacility/bioinformatics/irida/ria/web/projects/settings/ProjectAssociatedProjectsAjaxController.java

```diff
@@ -73,8 +83,7 @@ public ResponseEntity<AjaxResponse> removeAssociatedProject(@RequestParam long p
 			service.removeAssociatedProject(projectId, associatedProjectId, locale);
 			return ResponseEntity.ok(new AjaxSuccessResponse(""));
 		} catch (Exception e) {
```

Comment:
Please change this to a specific exception instead of the generic Exception as that will capture a lot more and potentially leak sensitive info.

Reply:
Updated in cb8ac126b830e6be2958a8fb2c65121b9d354d7c